src/transport_ofi.h

```diff
@@ -756,9 +755,19 @@ shmem_transport_atomic_nb(void *target, const void *source, size_t full_len,
 	uint64_t polled = 0;
         uint64_t key;
         uint8_t *addr;
+	size_t shmem_transport_ofi_max_atomic_size = 0;
```

Comment:
This variable is still using the naming convention for global variables.  Should be renamed to something like ```max_atomic_size``` to avoid confusion?